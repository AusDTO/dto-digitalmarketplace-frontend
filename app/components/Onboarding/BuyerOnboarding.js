import React from 'react'
import * as styles from './Onboarding.scss'

const BuyerOnboarding = () =>
  <div className="row">
    <div className="col-sm-push-2 col-sm-8 col-xs-12">
      <h1 title="Welcome to the Marketplace">Welcome to the Marketplace</h1>
      <p>How can we help you?</p>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>Read the Marketplace guides, contracts and compliance information</h4>
        </span>
        <span>
          <a href="https://marketplace1.zendesk.com/hc/en-gb/categories/115001542047-Buyer-guide-and-FAQs">
            Learn more
          </a>
        </span>
      </div>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>Request quotes, resumes or proposals</h4>
        </span>
        <span>
          <a href="/buyers">Post a brief</a>
        </span>
      </div>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>Need help writing a brief</h4>
        </span>
        <span>
          <a href="/contact-us">We are online now</a>
        </span>
      </div>
    </div>
  </div>

export default BuyerOnboarding
