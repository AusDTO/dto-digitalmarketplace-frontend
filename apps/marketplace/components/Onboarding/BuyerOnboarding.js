import React from 'react'

import styles from './Onboarding.scss'

const BuyerOnboarding = () => (
  <div className="row">
    <div className="col-sm-push-2 col-sm-8 col-xs-12">
      <h1 title="Welcome to the Marketplace" className={styles.contentHeading}>
        Welcome to the Marketplace
      </h1>
      <p>How can we help you?</p>
      <h2 className={styles.contentHeading}>Read the Marketplace guides, contracts and compliance information</h2>
      <span>
        <a href="https://marketplace1.zendesk.com/hc/en-gb/categories/115001542047-Buyer-guide-and-FAQs">Learn more</a>
      </span>
      <h2 className={styles.contentHeading}>Create a new opportunity for a digital specialist or outcome</h2>
      <span>
        <a href="/2/buyer-dashboard">Start a new brief</a>
      </span>
      <h2 className={styles.contentHeading}>Need help writing a brief?</h2>
      <span>
        All you need to do is <a href="https://marketplace1.zendesk.com/hc/en-gb/requests/new">contact us</a>
      </span>
    </div>
  </div>
)

export default BuyerOnboarding
