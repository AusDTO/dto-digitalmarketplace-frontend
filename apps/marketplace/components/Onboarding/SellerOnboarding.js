import React from 'react'
import { rootPath } from '../../routes'

const SellerOnboarding = () => (
  <div className="row">
    <div className="col-xs-12">
      <h1 className="au-display-xl" title="Welcome to the Marketplace">
        Welcome to the Marketplace
      </h1>
      <p>
        <a href={`${rootPath}/login`}>Log in</a> to set up your seller profile on the Marketplace.
      </p>
    </div>
  </div>
)
export default SellerOnboarding
