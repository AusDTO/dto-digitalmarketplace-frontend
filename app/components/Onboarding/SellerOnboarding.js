import React from 'react'
import * as styles from './Onboarding.scss'

const SellerOnboarding = () =>
  <div className="row">
    <div className="col-sm-push-2 col-sm-8 col-xs-12">
      <h1 title="Welcome to the Marketplace">Welcome to the Marketplace</h1>
      <p>What do you want to do next?</p>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>List my digital offering on the Marketplace</h4>
        </span>
        <span>
          <a href="/become-a-seller">Set up my shopfront</a>
        </span>
      </div>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>Apply for an opportunity listed on the Marketplace</h4>
        </span>
        <span>
          <a href="/digital-marketplace/opportunities">Browse opportunities</a>
        </span>
      </div>
      <div className={styles.onBoardingItemWrapper}>
        <span>
          <h4>Learn about adding case studies to your shopfront</h4>
        </span>
        <span>
          <a href="https://marketplace1.zendesk.com/hc/en-gb/articles/115011407668-Adding-case-studies">
            Adding a case study
          </a>
        </span>
      </div>
    </div>
  </div>

export default SellerOnboarding
