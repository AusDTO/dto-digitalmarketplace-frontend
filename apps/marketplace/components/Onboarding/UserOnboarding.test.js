import React from 'react'
import renderer from 'react-test-renderer'
import expect from 'expect'
import SellerOnboarding from './SellerOnboarding'
import BuyerOnboarding from './BuyerOnboarding'

jest.mock('shared/Icon/_getIcons')

describe('Test suite for UserOnboarding container component', () => {
  it('Because snaphots', () => {
    const component = renderer.create(<SellerOnboarding />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('Because snaphots', () => {
    const component = renderer.create(<BuyerOnboarding />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
