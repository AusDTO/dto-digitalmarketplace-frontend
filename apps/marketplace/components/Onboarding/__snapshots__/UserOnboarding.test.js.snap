// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suite for UserOnboarding container component Because snaphots 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-push-2 col-sm-8 col-xs-12"
  >
    <h1
      title="Welcome to the Marketplace"
    >
      Welcome to the Marketplace
    </h1>
    <p>
      What do you want to do next?
    </p>
    <h2
      className="au-display-lg"
    >
      List my digital offering on the Marketplace
    </h2>
    <span>
      <a
        href="/sellers/application"
      >
        Set up my shopfront
      </a>
    </span>
    <h2
      className="au-display-lg"
    >
      Apply for an opportunity listed on the Marketplace
    </h2>
    <span>
      <a
        href="/2/opportunities"
      >
        Browse opportunities
      </a>
    </span>
    <h2
      className="au-display-lg"
    >
      Learn about adding case studies to your shopfront
    </h2>
    <span>
      <a
        href="https://marketplace1.zendesk.com/hc/en-gb/articles/115011407668-Adding-case-studies"
      >
        Adding a case study
      </a>
    </span>
  </div>
</div>
`;

exports[`Test suite for UserOnboarding container component Because snaphots 2`] = `
<div
  className="row"
>
  <div
    className="col-sm-push-2 col-sm-8 col-xs-12"
  >
    <h1
      title="Welcome to the Marketplace"
    >
      Welcome to the Marketplace
    </h1>
    <p>
      How can we help you?
    </p>
    <h2
      className="au-display-lg"
    >
      Read the Marketplace guides, contracts and compliance information
    </h2>
    <span>
      <a
        href="https://marketplace1.zendesk.com/hc/en-gb/categories/115001542047-Buyer-guide-and-FAQs"
      >
        Learn more
      </a>
    </span>
    <h2
      className="au-display-lg"
    >
      Create a new opportunity for a digital specialist or outcome
    </h2>
    <span>
      <a
        href="/buyers"
      >
        Start a new brief
      </a>
    </span>
    <h2
      className="au-display-lg"
    >
      Need help writing a brief?
    </h2>
    <span>
      All you need to do is 
      <a
        href="https://marketplace1.zendesk.com/hc/en-gb/requests/new"
      >
        contact us
      </a>
    </span>
  </div>
</div>
`;
